### PR TITLE
Add images to the Live Bidding UI

### PR DESCRIPTION
**Pull request by orta**
Adds aspect ratio + the real image data, punted on the animations, wanted to focus on something else for a bi.

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Errors</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>No CHANGELOG changes made</td>
    </tr>
        <tr>
      <td>:no_entry_sign:</td>
      <td>Developer Specific file shouldn&#39;t be changed</td>
    </tr>
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>Needs testing on a Phone if change is non-trivial</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARArtworkFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkFavoritesNetworkModelSpec</a> in <code>test_getFavorites__makes_request_if_no_request_is_in_progress</code></li>
<li><a href="https://github.com/search?q=ARArtworkViewControllerBuyButton+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARArtworkViewControllerBuyButtonSpec</a> in <code>test_buy_button__posts_order_if_artwork_has_no_edition_sets</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=ARCreateAccountViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARCreateAccountViewControllerSpec</a> in <code>test_warning_view__displays_a_message</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__shows_favorite_and_follow_buttons_for_logged_in_user</code></li>
<li><a href="https://github.com/search?q=AREmbeddedModelPreviewViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">AREmbeddedModelPreviewViewControllerSpec</a> in <code>test_iPhone_peek__passes_artwork_to_share_controller</code></li>
<li><a href="https://github.com/search?q=ARFairViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARGeneViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARGeneViewControllerSpec</a> in <code>test_creates_an_NSUserActivity</code></li>
<li><a href="https://github.com/search?q=ARTopMenuViewController+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARTopMenuViewControllerSpec</a> in <code>test_navigation__favorites__logged_in__tab_behavior__already_selected__pops_to_root</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by orta** [comment]
screen shot 2016-03-15 at 4 11 13 pm

This isn't perfect but it's good enough for review @ashfurrow

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionSalesPerson.swift [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
     func lotViewModelForIndex(index: Int) -> LiveAuctionLotViewModel?
+    func lotViewModelRelativeToShowingIndex(offset: Int) -> LiveAuctionLotViewModel?
 
     // Remove me
     func setupWithStub()
 }
 
-class LiveAuctionsSalesPerson:  NSObject, LiveAuctionsSalesPersonType {
-    var currentIndex = 0
+class LiveAuctionsSalesPerson:  NSObject, LiveAuctionsSalesPersonType, UIPageViewControllerDelegate {
```
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionSalesPerson.swift]: Not totally sold on having the salesperson do UI-stuff, even just as a delegate (as you say above). Any way we could put the delegate in an `extension LiveAuctionsSalesPerson: UIPageViewControllerDelegate` to make it easy to see which methods are necessary, maybe to move out later?

Comment (orta) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionSalesPerson.swift]: Yeah think it makes sense to vend that out into another object, and DI in the signal

**Comment by ashfurrow** [comment]
Looks cool, would like to see a snapshot test showing the images laid out, even if they're just grey boxes.

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionSalesPerson.swift (hunk 1 above)
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionSalesPerson.swift]: Sounds fab

**Comment by orta** [comment]
OK, this is green

**Pull request merged**